src/main/java/com/pi4j/devices/bme280/README.md

```diff
@@ -1,7 +1,24 @@
 https://pdf1.alldatasheet.com/datasheet-pdf/view/1132060/BOSCH/BME280.html
 
-The program uses org.slf4j.simple Class SimpleLogger as suggested by the Pi4j_V2 documentation. Consult that logger
-class documentation to understand its various logging options.
+  __  __ _               _          _ 
```

Comment:
I am not opposed to some nice ascii art, but I'd prefer if we'd limit it to the top level readme to avoid repetition?